src/wire/libevent_server.cpp

```diff
@@ -45,11 +48,14 @@ void LibeventServer::CreateNewConn(const int &connfd, short ev_flags,
                                    LibeventThread *thread,
                                    ConnState init_state) {
   auto &global_socket_list = GetGlobalSocketList();
+  recent_connfd = connfd;
```

Comment:
Yes, this is a problem. We are currently using this variable only in single thread test to get an active connection and we will try to use other approaches to get correct recent_connfd in multi-thread test, thanks.